testing/resources/istio-ingressgateway-patch.yaml

```diff
@@ -0,0 +1,5 @@
+spec:
+  ports:
+  - name: http2
+    nodePort: 31280
```

Comment:
Is this to support newer versions of Istio?

Reply:
yes